File: Domain/Entities/Statistics.cs
namespace Kaizen.Domain.Entities
{
    public abstract class Statistics
    {
        public int AppliedActivities { get; set; }
        public int ClientsVisited { get; set; }
        public int ClientsRegistered { get; set; }
        public decimal Profits { get; set; }
    }
}

File: Kaizen/ClientApp/src/app/modules/statistics/models/statistics.ts
export interface Statistics {
  appliedActivities: number;
  clientsVisited: number;
  clientsRegistered: number;
  profits: number;
}

export interface DayStatistics extends Statistics {
  id: number;
  date: Date;
}

export interface MontStatistics extends Statistics {
  id: number;
  date: Date;

  dayStatistics?: DayStatistics[];
}

export interface YearStatistics extends Statistics {
  id: number;
  year: number;

  montStatistics?: MontStatistics[];
}
